haskell/gtk-gui/stack.yaml

```diff
@@ -0,0 +1,5 @@
+resolver:
+  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/22/11.yaml
```

Comment:
I'll use the `resolver: lts-21.13` instead, is more commonly used across haskell projects.